index.html

```diff
@@ -653,6 +664,35 @@ <h2>Exposing change of MediaStreamTrack configuration</h2>
       </div>
     </p>
     </div>
+    <h3>ConfigurationChangeEvent Interface</h3>
+    <div>
+      <pre class="idl">
+[Exposed=Window]
+interface ConfigurationChangeEvent : Event {
+  constructor(DOMString type, ConfigurationChangeEventInit changeEventInitDict);
+  readonly attribute FrozenArray&lt;DOMString&gt; changes;
+};
+
+dictionary ConfigurationChangeEventInit : EventInit {
+  required sequence&lt;DOMString&gt; changes;
+};</pre>
+    <p>The <dfn data-dfn-for="ConfigurationChangeEvent">changes</dfn> getter
+      steps are to return the value that the corresponding attribute was
+      initialized to.
+    </p>
+    </div>
+    <h3>Example</h3>
+    <div>
+      <p>This example shows how to monitor external background blur changes.</p>
+      <pre class="example">
+const stream = await navigator.mediaDevices.getUserMedia({ video: true });
+const [track] = stream.getVideoTracks();
+track.addEventListener("configurationchange", (event) => {
+  if (event.changes.includes("backgroundBlur")) {
+    // Background blur configuration has changed.
```

Comment:
Nit, and completely up to you, but maybe a clearer example would also indicate why an app might care about this? Maybe the comment could mention it would toggle its own blurring to be the inverse, so as to always be blurred but never double-blurred?